schedule/yam/test_cases/select_modules_and_patterns+registration.yaml

```diff
@@ -32,7 +32,6 @@ schedule:
     - console/install_packages_simple
     - console/scc_deregistration
     - console/firewalld_add_port
-    - console/setup_libyui_running_system
-    - x11/addon_products_via_SCC_yast2
+    - console/suseconnect_scc
```

Comment:
Just wondering why replace the above two test modules with this one?

Reply:
to make it run on spvm.

Reply:
@chcao  @lemon-suse  can you please take a look again? thanks!